i3lock.1

```diff
@@ -85,6 +86,11 @@ show you the current PAM state (whether your password is currently being
 verified or whether it is wrong).
 
 .TP
+.B \-C, \-\-capslock-indicator
+Enable the capslock indicator. This will give visual feedback on the state of
+the caps lock mod.
```

Comment:
Please improve the docs: describe how the visual indicator looks like, i.e. how the user can tell if caps lock is on. Also describe the motivation behind it, i.e. that the computer cannot be unlocked while caps lock is on, unless you are aware of it.